.pipelines/.vsts-vhd-automation.yaml

```diff
@@ -2,17 +2,19 @@ pool:
   vmImage: ubuntu-latest
 
 steps:
+- bash: |
+        echo $ADO_PAT | az devops login --organization=https://dev.azure.com/msazure
```

Comment:
This does not actually log the PAT, pipeline run for reference: https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=48614630&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=76467409-274e-589c-5c88-4286504c504b